### DISCUSSION TITLE
Module parse failed: Unexpected token

### DISCUSSION THREAD

**Discussion by samartsevigor** [WalletConnect/walletconnect-monorepo#1594] [Developer Support]
<img width="1045" alt="Снимок экрана 2022-11-03 в 19 34 22" src="https://user-images.githubusercontent.com/63883555/199805978-669b52a3-fbee-4f0e-b765-3fdedde0f544.png">

installed the @web3modal/react library and wanted to use it, but I get this story. Tried different ways with stackoverflow. Was added to package.json

```
  "babel": {
    "presets": [
      "@babel/preset-env"
    ]
  },
```

I tried to add presets to Babel's config, no way ...

```
module.exports = function (api) {
  api.cache(true);
  return {
    "presets": ["@babel/preset-env","@babel/preset-react"],
    plugins: [
      [
        'formatjs',
        {
          idInterpolationPattern: '[sha512:contenthash:base64:6]',
          ast: true,
        },
      ],
    ],
  };
};

```

**Comment by xzilja** [comment]
It's erroring due to ES2020 syntax that we are targeting, in particular the nullish coalescing bit `??`

I believe babel's latest version supports this out of the box, but if you need to support it in earlier ones there are loaders like
https://babeljs.io/docs/en/babel-plugin-proposal-nullish-coalescing-operator

You will likely need few other loaders for optinoal chaining perhaps
